### PR TITLE
add basic sops config

### LINKED ISSUE #30: we need a secure way to share secrets
While continuing to refactor I'm going to need to be able to use the Github and Cloudflare APIs. I see some sensitive values have variable definitions. I'm guessing these are passed on the command line or env vars.

If so then the simplest thing to do might be to use [sops](https://getsops.io/docs/] to commit an encrypted `.env` file to this repo. Sops supports using a key stored in AWS KMS to encrypt files which solves the bootstrapping problem. We've already both got secure access to AWS.

If you're up for this @IanEdington I can cut a feature branch and get sops initialized, then you could populate it.

### PR DESCRIPTION
**Pull request by rsalmond**
This change adds a basic [sops](https://getsops.io/) config file and a corresponding encrypted env file. This configures sops to use an AWS KMS key that @IanEdington and myself have IAM access to both administer (rotate, etc) and use for sops encrypt/decrypt operations. Since the whole point of this exercise is to support terraform changes, I haven't bothered adding the KMS key to terraform yet but that will come in a future PR.

I have omitted instructions for how to use the secrets as I'm not sure yet what the best way is. TBD on a future TF PR.

aside: I strongly recommend we all create an [aws profile](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-format) called `gpo` instead of exporting env vars so we can reference them in tooling (see line 4 of `sops.yaml`. Later when we have stage / prod isolation, our tooling cannot accidentally talk to the wrong environment by someone exporting the wrong env vars.

tl;dr - put this in `~/.aws/credentials` and `chmod 600`.

```
[gpo]
aws_access_key_id=<your key id>
aws_secret_access_key=<your secret key>
```

Fixes #30

### CODE REVIEW

**Comment by IanEdington** [comment]
I've added the secrets from the current `terraform.tfvars` file on my local machine. I have to say I had a moment of hesitation before pushing secrets to a public repo

**Review by IanEdington — APPROVED** [review]
